docs/userguide/networking/dockernetworks.md

```diff
@@ -0,0 +1,368 @@
+9f904ee27bf5        none                null
+cf03ee007fb4        host                host
+```
+
+Historically, these three networks are  part of Docker's implementation. When you run a container you can use the `--net` flag to specify which network you want to run a container on. These three networks are still available to you.
+
+The `bridge` network represents the `docker0` network present in all Docker installations. The Docker daemon launches each new container this network by default. You can see this bridge as part of the hosts network stack by doing an `ifconfig` on the host.
+
+```
+ubuntu@ip-172-31-36-118:~$ ifconfig
+docker0   Link encap:Ethernet  HWaddr 02:42:47:bc:3a:eb  
+          inet addr:172.17.0.1  Bcast:0.0.0.0  Mask:255.255.0.0
+          inet6 addr: fe80::42:47ff:febc:3aeb/64 Scope:Link
+          UP BROADCAST RUNNING MULTICAST  MTU:9001  Metric:1
+          RX packets:17 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:8 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:1100 (1.1 KB)  TX bytes:648 (648.0 B)
+...
+
+The `none` network adds a container to a container-specific network stack.  That container lacks a network interface. Attaching to such a container and looking at it's stack you see this:
+
+```
+ubuntu@ip-172-31-36-118:~$ docker attach nonenetcontainer
+
+/ # cat /etc/hosts
+127.0.0.1	localhost
+::1	localhost ip6-localhost ip6-loopback
+fe00::0	ip6-localnet
+ff00::0	ip6-mcastprefix
+ff02::1	ip6-allnodes
+ff02::2	ip6-allrouters
+/ # ifconfig
+lo        Link encap:Local Loopback  
+          inet addr:127.0.0.1  Mask:255.0.0.0
+          inet6 addr: ::1/128 Scope:Host
+          UP LOOPBACK RUNNING  MTU:65536  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
+
+/ #
+```
+**Note**: You can detach from the container and leave it running with `CTRL-p CTRL-q`.
+
+The `host` network adds a container on the hosts network stack. You'll find the network configuration inside the container is identical to the host.
+
+With the exception of the the `bridge` network, you really don't need to interact with these default networks. While you can list and inspect these networks, you cannot remove them. They are required by your Docker installation.  
+However, you can add to the list of networks by creating new ones yourself. Before you learn more about creating your own networks, it is worth looking at the `default` network a bit.
+
+
+### The default bridge0 network in detail
+The default `bridge` network is present on all Docker hosts. First, inspect the `bridge` network:
+
+```
+$ sudo docker network inspect bridge
+{
+    "name": "bridge",
+    "id": "7fca4eb8c647e57e9d46c32714271e0c3f8bf8d17d346629e2820547b2d90039",
+    "driver": "bridge",
+    "containers": {}
```

Comment:
The response has changed in rc4, we now capitalize all JSON elements for network API responses, to be consistent with other responses in the API (e.g. `id` now is `Id`)